packages/proxy/lib/http/util/prerequests.ts

```diff
@@ -320,4 +320,14 @@ export class PreRequests {
     this.pendingRequests = new QueueMap<PendingRequest>()
     this.pendingUrlsWithoutPreRequests = new QueueMap<PendingUrlWithoutPreRequest>()
   }
+
+  private tryDecodeURI (url: string) {
```

Comment:
This could be a static method right? Or external to the class?

Reply:
yep, moved it, [43456a6](https://github.com/cypress-io/cypress/pull/28522/commits/43456a68294f9da935cf76e68bd4c775afc9ca25)